src/components/EditionCard/EditionCard.tsx

```diff
@@ -48,63 +51,89 @@ export const EditionCard: React.FC<{ edition: WorkEdition; title: string }> = ({
-        src: coverUrl,
-        alt:
-          coverUrl === PLACEHOLDER_COVER_LINK
-            ? "Placeholder Cover"
-            : `Cover for ${EditionCardUtils.editionYearText(edition)}`,
-        size: "xsmall",
-        aspectRatio: "original",
-      }}
-      isCentered
-      isBordered
-      isAlignedRightActions
-      p="s"
-      flexFlow={{ md: "column nowrap", lg: "row" }}
-      alignItems={{ md: "flex-start", lg: "center" }}
-      sx={{ ".card-right": { maxWidth: { base: "100%", md: "200px" } } }}
+    <Box
+      border="1px"
+      borderColor="ui.border.default"
+      padding="s"
+      paddingBottom="l"
+      paddingRight="l"
     >
-      <CardHeading
-        level="h3"
-        size="heading6"
-        id="stack1-heading1"
+      <Flex gap="xs" flexDirection={{ base: "column", lg: "row" }}>
+        {isPhysicalEdition && <CardRequiredBadge />}
+        {isFeaturedEdition && <FeaturedEditionBadge />}
+      </Flex>
+      <Card
+        id={`card-${edition.edition_id}`}
+        layout="row"
+        imageProps={{
+          src: coverUrl,
+          alt:
+            coverUrl === PLACEHOLDER_COVER_LINK
+              ? "Placeholder Cover"
+              : `Cover for ${EditionCardUtils.editionYearText(edition)}`,
+          size: "xsmall",
+          aspectRatio: "original",
+        }}
+        isAlignedRightActions
+        paddingTop="m"
+        flexFlow={{ md: "column nowrap", lg: "row" }}
+        justifyContent={{ md: "center", lg: "left" }}
         sx={{
-          span: {
-            fontSize: "18px",
-            a: {
-              textDecoration: "none",
-            },
+          ".card-right": {
+            maxWidth: { base: "100%", lg: "200px" },
+            marginStart: { base: "0", lg: "m" },
+            marginTop: { base: "xs", lg: 0 },
           },
         }}
       >
-        <Flex alignItems="center">
-          <span>{editionYearElem(edition)}</span>
-          {isPhysicalEdition && <PhysicalEditionBadge />}
-        </Flex>
-      </CardHeading>
-      <CardContent>
-        <Box>
-          {EditionCardUtils.getPublisherAndLocation(
-            edition.publication_place,
-            edition.publishers
+        <CardHeading
+          level="h3"
+          size="heading6"
+          id="stack1-heading1"
+          sx={{
+            span: {
+              fontSize: "18px",
+              a: {
+                textDecoration: "none",
+              },
+            },
+          }}
+        >
+          <Flex alignItems="center" gap="xs">
+            <span>{editionYearElem(edition)}</span>
+            {isPhysicalEdition && <PhysicalEditionBadge />}
+          </Flex>
+        </CardHeading>
+        <CardContent>
+          <Box>
+            {EditionCardUtils.getPublisherAndLocation(
+              edition.publication_place,
+              edition.publishers
+            )}
+            <Box>{EditionCardUtils.getLanguageDisplayText(edition)}</Box>
+            <Link to="/license">
```

Comment:
Let's refactor this into a React component instead of a function that returns JSX